estudios_socioeconomicos/views.py

```diff
@@ -0,0 +1,44 @@
+from django.shortcuts import render, get_object_or_404
+from django.contrib.auth.decorators import login_required, user_passes_test
+
+from captura.utils import get_study_info
+from perfiles_usuario.utils import is_capturista_or_administrador, is_capturista
+from familias.models import Integrante
+from familias.utils import total_egresos_familia, total_ingresos_familia, total_neto_familia
+from indicadores.models import Transaccion, Ingreso
+from .models import Estudio, Foto
+
+
+@login_required
+@user_passes_test(is_capturista_or_administrador)
+def focus_mode(request, id_estudio):
+    """ View to see the detail information about a family and their study.
+    """
+    context = {}
+
+    estudio = get_object_or_404(Estudio.objects.filter(pk=id_estudio))
+
+    if is_capturista(request.user):
```

Comment:
kind of unrelated, but i was trying to think if we could do this in the decorator since we need to actually to do this for every view that is accessible for capturistas am i right?